libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/tooltip/ElementTooltipDefaults.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.designsystem.components.tooltip
+
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.TooltipDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.IntOffset
+import androidx.compose.ui.unit.IntRect
+import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.unit.LayoutDirection
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.PopupPositionProvider
+
+object ElementTooltipDefaults {
+    /**
+     * [PopupPositionProvider] designed for use with a [PlainTooltip].
+     *
+     * @param spacingBetweenTooltipAndAnchor the spacing between the tooltip and the anchor.
+     * @param windowPadding the padding between the tooltip and the edge of the window.
+     *
+     * @return a [PopupPositionProvider].
+     */
```

Comment:
That's great, though it took my quite a while to understand why you had created this one instead of using M3's.
In such cases it's great to begin the docstring with something like:

"Wrapper around `TooltipDefaults.rememberPlainTooltipPositionProvider()` which allows to add padding between the edge of the window and the tooltip."

Reply:
Agreed. I've updated the docs (https://github.com/vector-im/element-x-android/pull/1710/commits/c999f792d8a1ca359296662454cb84e62693b60a) and hopefully that is a bit clearer!
https://github.com/vector-im/element-x-android/pull/1710/commits/c999f792d8a1ca359296662454cb84e62693b60a